### ISSUE TITLE
LogLoss function?

### ISSUE THREAD

**Issue by roger77** [Microsoft/CNTK#1674]
I am not able to find the LogLoss function, is it implemented?

**Comment by DanielMerget** [comment]
You probably mean cross entropy?

https://github.com/Microsoft/CNTK/wiki/Loss-Functions-and-Metrics

**Comment by roger77** [comment]
I think I mean `binary_cross_entropy` as in [https://github.com/Microsoft/CNTK/blob/2e246dbe8b8d5e47f8f825ce9922fdf40446e652/bindings/python/cntk/losses/__init__.py](url)

but when I substitute this function instead of `cross_entropy_with_softmax` I get a function not found error.

**Comment by DanielMerget** [comment]
From the commit log it seems this particular function was only added 29 days ago. You probably need to update to a newer version if you want to use it.

**Comment by n17s** [comment]
Also don't forget that cross_entropy_with_softmax expects as inputs logits (i.e. unconstrained values) while binary_cross_entropy expects as input a probability (i.e. the output of a sigmoid)

Closing this as it's been a while. Feel free to reopen with questions.
